packages/autorest.python/autorest/codegen/templates/serialization.py.jinja2

```diff
@@ -2002,6 +2002,8 @@ class Deserializer(object):
         if isinstance(attr, ET.Element):
             attr = int(attr.text)  # type: ignore
         try:
+            if isinstance(attr, str):
+                attr = int(attr)
```

Comment:
@lmazuel what are your thoughts on this? I think it looks good. If you try to call `int` on something that can't be cast to an int, i.e. `int("blah")`, you get a ValueError, which will get caught on 2008

Reply:
I would just do `int(attr)`. We don't need the `isintance`. If it's already an int, it's a no-op, and if not, either the conversion works or as you said, ValueError in 2008

Reply:
good point, thanks!

Reply:
@iscai-msft Updated.